include/ofi_util.h

```diff
@@ -801,10 +801,10 @@ struct util_eq {
 
 struct util_event {
 	struct slist_entry	entry;
-	int			size;
+	ssize_t			size;
 	int			event;
 	int			err;
-	uint8_t			data[0];
+	uint8_t			data[0]; /* should be 8-byte aligned */
```

Comment:
We don't actually need 8-byte alignment here, correct?  Just pointer alignment?

Reply:
Pointer is a bit confusing I guess, because with struct util_event data[] is a not a pointer. I added "offset".